### ISSUE TITLE
Extra sound after water balances

### ISSUE THREAD

**Issue by marlitas** [phetsims/mean-share-and-balance#327]
As I was checking the built version for the RC I noticed that there is an extra sound emerging after water is balanced in the Level Out Screen.

https://github.com/user-attachments/assets/4991fa87-1167-4cb3-adf3-e15825cdb712

I am assigning this to @jbphet because I believe he will have a better idea of where to start. Please let me know if you would like some support.

**Comment by marlitas** [comment]
Once I updated Chrome this stopped happening and both @jbphet and I are unable to reproduce consistently. We hear it happen very randomly but it is a minor amount of times, and because it is not repetitive we do not feel it detracts from user experience. 

We both spent about 1 timeboxed hour to pair on further exploration, and after the hour determined it was not worth looking into it further with the current info we have. We would like to keep this open for QA to keep an ear out for it, but otherwise are not concerned.

**Comment by marlitas** [comment]
This came back after @jbphet and I paired this morning. I am seeing the behavior both in a build in the RC branch as well as locally. It seems to only be occurring on macs in either Chrome or Firefox. 

I'm not sure I got to the true heart of the issue, but it has really sucked up a lot of time now and I think we are at the point where we can cut the feature we've been fighting up against. 

In https://github.com/phetsims/mean-share-and-balance/issues/227 we introduced a feature in the step function that would eagerly set the water levels of cups if they got close enough to their final target. This helped clean up some processes running in the background and was most notable for users that may use screen readers, or phet-io clients tracking the water level values. However, eagerly setting the water level value is now causing some quirks in the sound generator where the water balance sound starts to play again. 

I looked at how the deviations and rounding intervals were interacting here to find a middle ground, and the only fix that I have been able to find after working on this for over an hour is to not set the water levels eagerly. I have decided to remove that logic in the code and I do not believe it will cause any regression in the behavior of the sim.

That said... QA should most definitely still keep an eye on water levels as they move between cups, especially in regards to phet-io state handling.
